applications/asset_tracker_v2/src/cloud/cloud_codec/aws_iot/aws_iot_codec.c

```diff
@@ -319,6 +320,16 @@ int cloud_codec_encode_data(struct cloud_codec_data *output,
 		goto add_object;
 	}
 
+	err = json_common_impact_data_add(rep_obj, impact_buf,
```

Comment:
Looks like this function is called twice, here and line `383`

Reply:
I removed json_common_imp_data_add now. No idea where that came from.